device-posture/device-posture.https.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+//<script src="../resources/orientation-event-helpers.js"></script>
+<script>
+'use strict';
+
+async function cleanup() {
+  await test_driver.set_device_posture('continuous');
+}
+
+promise_test(async (t) => {
+  t.add_cleanup(cleanup);
+  const promise = new Promise(resolve => {
+    navigator.devicePosture.addEventListener("change", () => {
+      resolve();
+    });
+  });
+
+  const postures = ['folded', 'continuous', 'folded'];
+  for (let posture of postures) {
```

Comment:
```suggestion
  for (const posture of postures) {
```

Reply:
Fixed in https://github.com/JuhaVainio/WPT/pull/4/commits/d2c3a71b9a1408a4c661ef228308296479937607